go.sum

```diff
@@ -1071,7 +1085,12 @@ golang.org/x/tools v0.0.0-20210106214847-113979e3529a/go.mod h1:emZCQorbCU4vsT4f
 golang.org/x/tools v0.1.0/go.mod h1:xkSsbof2nBLbhDlRMhhhyNLN/zl3eTqcnHD5viDpcZ0=
 golang.org/x/tools v0.1.2 h1:kRBLX7v7Af8W7Gdbbc908OJcdgtK8bOz9Uaj8/F1ACA=
 golang.org/x/tools v0.1.2/go.mod h1:o0xws9oXOQQZyjljx8fwUC0k7L1pTE6eaCbjGeHmOkk=
+<<<<<<< HEAD
```

Comment:
@nitishSr There are still rebase bits here that need to be removed. For `go.sum` I recommend just checking it out and letting it get regenerated.

so:
```
git checkout go.sum
go mod tidy
```